### PR TITLE
Update index.rst - update version number

### PR DESCRIPTION
**Pull request by muescha**
update version number to 1.0.0

- used static text at first (01849a126e604d1094fe0283102ecdccce83c1b8)
- then changed to variable (1005f75d27c752ba42fccbd0a097db4ccf58ef83) - but i don't know if a variable works

### CODE REVIEW

**Comment by muescha** [comment]
- used static text at first (01849a126e604d1094fe0283102ecdccce83c1b8)
- then changed to variable (1005f75d27c752ba42fccbd0a097db4ccf58ef83) - but i don't know if a variable works

**Comment by arichard4** [comment]
I don't think that the variable works. I tested locally with sphinx-build and it output the text "{{ version }}". Let me investigate further, the variable approach would be nice; however, readthedocs.io, the most frequent place the docs are stored, mentions the version beside the docs, so we can just remove that if needed.

**Comment by arichard4** [comment]
@muescha According to the [sphinx docs](https://sphinx-doc-zh.readthedocs.io/en/latest/markup/inline.html#other-semantic-markup), you can insert the version with "|version|". While you're here, please move the version number to be after the word version. The can be tested with "sphinx docsrc doc", and then opening doc/index.html

**Review by arichard4 — CHANGES_REQUESTED** [review]
Pls change to "version |version|"

File: docsrc/index.rst [hunk 1]
```diff
@@ -11,4 +11,4 @@ Contents:
    inline
    module
 
-This is documentation for 0.26.0 version of `Luacheck <https://github.com/lunarmodules/luacheck/>`_, a linter for `Lua <https://www.lua.org/>`_.
+This is documentation for {{ version }} version of `Luacheck <https://github.com/lunarmodules/luacheck/>`_, a linter for `Lua <https://www.lua.org/>`_.
```
Comment (alerque) [inline docsrc/index.rst:14]:
```suggestion
This is documentation for version |version| of `Luacheck <https://github.com/lunarmodules/luacheck/>`_, a linter for `Lua <https://www.lua.org/>`_.
```

**Review by alerque — APPROVED** [review]
Thanks for catching this and taking the time to contribute. I tested @arichard4's suggested variable version locally and it seems to work, so went ahead and ammended this PR with that info. Thanks to both of you for tracking that down.

**Pull request merged**